challenge/files/09_slices.toml

```diff
@@ -0,0 +1,56 @@
+name = "Slice Append Behavior"
+category = "wtf"
+default_code_snippet = """
+func main() {
+	var x []int
+	x = append(x, 0)
+	x = append(x, 1)
+	x = append(x, 2)
+
+	fmt.Println(append(append(x, 3), append(x, 4)...))
+}
+"""
+learning_advise = """
+Learn about how slices work in Go, especially how the append function can affect slice capacity and length.
+"""
+
+[[learning_links]]
+title = "Go Slices: Usage and Internals"
+url = "https://go.dev/blog/slices-intro"
+
+[[learning_links]]
+title = "Go by Example: Slices"
+url = "https://gobyexample.com/slices"
+
+[[learning_links]]
+title = "Learn Go With Tests: Arrays and Slices"
+url = "https://quii.gitbook.io/learn-go-with-tests/go-fundamentals/arrays-and-slices"
+
+[[questions]]
+text = "Select program output"
+type = "select_answers"
+answers = [
+    { text = '[0 1 2 3 0 1 2 4]' },
+    { text = '[0 1 2 4 0 1 2 4]' , code_line_ranges = [] },
+    { text = "panic: slice bounds out of range" },
+    { text = "Other" },
+]
+
```

Comment:
Let's add few more questions:

1. Something like "What is slice?"
2. "Choose x variable slice capacity?" - Answers: 1,2,4,8 - this question to check understanding of slice grow